internal/provider/atlas_migration_data_source.go

```diff
@@ -144,7 +145,7 @@ func (d *MigrationDataSource) Read(ctx context.Context, req datasource.ReadReque
 		return
 	}
 	r, err := d.client.MigrateStatus(ctx, &atlas.MigrateStatusParams{
-		ConfigURL: fmt.Sprintf("file://%s", cfgPath),
+		ConfigURL: fmt.Sprintf("file://%s", strings.ReplaceAll(cfgPath, `\`, "/")),
```

Comment:
Do you know https://pkg.go.dev/path/filepath#ToSlash?

Reply:
I didn't, perfect! I shall update the PR...